README.md

```diff
@@ -65,11 +65,14 @@ This run script deploys React + Flask. See the run script for individual gcloud
 ```
 and `gcloud auth login` if it asks you to authenticate, or insert YubiKey.
 
-#### How to Deploy to Staging
+## Deploy to Staging
```

Comment:
@sdzhong new section on how to deploy to Staging (or whatever you want to call it in your yaml's and .env file)